src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise1
+  .then(result => {
+    const message = createMessage(result);
+
+    document.body.append(message);
+  });
+
+promise2
+  .catch(result => {
+    const message = createMessage(result);
```

Comment:
Replace 'result' with 'error' for better readability in the catch block, as it represents an error.